plugins/janus_audiobridge.c

```diff
@@ -1047,6 +1062,9 @@ static struct janus_json_parameter join_parameters[] = {
 	{"rtp", JSON_OBJECT, 0},
 	{"secret", JSON_STRING, 0}
 };
+static struct janus_json_parameter record_parameters[] = {
+	{"record", JANUS_JSON_BOOL, JANUS_JSON_PARAM_REQUIRED}
```

Comment:
Since there's `record_file` too, now, it should be validates as well.

Reply:
record_file is optional parameter, if user do not supply parameter we are using default name. Do we still need to add it for validation ?

Reply:
ALL parameters need to be validated, optional or not. You'll see that we have a lot of parameters defined without `JANUS_JSON_PARAM_REQUIRED`, for instance.